com.unity.render-pipelines.core/Editor/RenderPipelineConversion.cs

```diff
@@ -0,0 +1,18 @@
+namespace UnityEditor.Rendering
+{
+    /// <summary>
+    /// A class for different conversions.
+    /// </summary>
+    public abstract class RenderPipelineConversion
```

Comment:
Hmm, I don't think this is different enough from `RenderPipelineConverter`. Needs a different name.

Reply:
Done